tests/impl.cpp

```diff
@@ -3502,7 +3502,23 @@ result_t test_mm_srli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int64_t *_b = (const int64_t *) impl.mTestIntPointer2;
+#if defined(__clang__)
```

Comment:
Please add the comment based on [this](https://github.com/DLTcollab/sse2neon/pull/129#discussion_r502309433).

Reply:
After that, the pull request should be merged.